Add welcome message for codespaces (#643)
primer/view_components: .devcontainer/welcome-message.txt

👋 Welcome to developing primer/view_components in codespaces!

👢 Your environment is all set up and bootstrapped.

> 🚀 To get started, start the server with the VS code task (⇧⌘B) or run

Reviewer: ```suggestion
🚀 To begin, start the server with the VS code task (⇧⌘B) or run
```

>    script/dev. This will start the Rails server at http://localhost:4000,

Reviewer: ```suggestion
   script/dev, which will start the Rails server at http://localhost:4000,
```

>    Storybook at http://localhost:5000 and docs at  http://localhost:5400.

Reviewer: ```suggestion
   Storybook at http://localhost:5000 and documentation at http://localhost:5400.
```

> ❔ Having other troubles? Open an issue in the repo for support!

Reviewer: ```suggestion
❔ Having other problems? Open an issue in the repo for support!
```
